tests/unit/amo/middleware/test_cspMiddleware.js

```diff
@@ -27,7 +35,42 @@ describe(__filename, () => {
   });
 
   describe('CSP Config', () => {
-    for (const env of deployedEnvs) {
+    for (const env of deployedEnvsWithStaticsOnSameDomain) {
+      // eslint-disable-next-lint no-loop-func
+      it(`should have a source-list config for ${env}`, () => {
```

Comment:
would something like this work instead?

```suggestion
    it.each(deployedEnvsWithStaticsOnSameDomain)(`should have a source-list config for ${env}`, (env) => {
```